.github/workflows/insider-linux.yml

```diff
@@ -132,6 +132,9 @@ jobs:
         - vscode_arch: arm64
           npm_arch: arm64
           image: vscodium/vscodium-linux-build-agent:buster-arm64
+        - vscode_arch: pp64le
+          npm_arch: ppc64le
+          image: vscodium/vscodium-linux-build-agent:bionic-ppc64le
```

Comment:
Can you provide a PR to add the `bionic-ppc64le` image?

Reply:
I opened PR VSCodium/vscode-linux-build-agent#15, was that not the right repo?